test/libyul/yulOptimizerTests/loopInvariantCodeMotion/move_storage_function.yul

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
This test seems to be the same as the one above

Reply:
Thanks. I changed `extcodesize()` to `sload()`.